sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosConflictProperties.java

```diff
@@ -11,7 +11,7 @@
 /**
  * The type Cosmos conflict properties.
  */
-public final class CosmosConflictProperties extends ResourceWrapper{
```

Comment:
`getId` is a required method in the Properties classes that we removed.

Reply:
please add back

Reply:
should I add back all this method?
getId
setId
getResourceId
setResourceId
getSelfLink
setSelfLink
getTimestamp
setTimestamp
getEtag
setEtag
setAltLink
getAltLink
validateResource

Reply:
added the following change based on the discussion:
add getId(), getResourceId, getEtag(), getTimestamp() to the following classes: CosmosConflictPropertiesCosmosContainerProperties
CosmosDatabaseProperties
CosmosPermissionProperties
CosmosStoredProcedureProperties
CosmosTriggerProperties
osmosUserDefinedFunctionProperties
CosmosUserProperties
ThroughputProperties

setId() is added if it is not existed.